pkg/ssh/pool.go

```diff
@@ -54,6 +56,33 @@ func startPool(ctx context.Context, serverAddr string, config *ssh.ClientConfig)
 	return p, nil
 }
 
+func retryNewClientConn(ctx context.Context, c net.Conn, addr string, conf *ssh.ClientConfig) (ssh.Conn, <-chan ssh.NewChannel, <-chan *ssh.Request, error) {
+	ticker := time.NewTicker(300 * time.Millisecond)
+	to := config.GetTimeout() / 6 // 5 seconds
```

Comment:
could we just keep it for the full 30s ? it's the worst case scenario right?

Reply:
it is better to fail fast. the `ssh` server starts very fast, but I have found the first connection to fail a few times